CONTRIBUTING.md

```diff
@@ -1,32 +0,0 @@
-# CONTRIBUTING
-
```

Comment:
Oh let's not delete this but instead let's modify it to fit this project. CONTRIBUTING.md files tell people how they could contribute to this project if they want to so they are important for open source projects.